Voice/Weather Phone IVR - Node.js/index.js

```diff
@@ -0,0 +1,175 @@
+require("dotenv").config();
+const parsePhoneNumber = require("libphonenumber-js");
+const axios = require("axios");
+let sw = require("@signalwire/realtime-api");
+let { Voice, Messaging } = sw;
```

Comment:
Just for style consistency, can we use `const` whenever we don't need to reassign the variable, and make this one line? Like

```js
const { Voice, Messaging } = require("@signalwire/realtime-api");
```